upgrades/upgradevalidation/migrate.go

```diff
@@ -9,23 +9,23 @@ import (
 	environscloudspec "github.com/juju/juju/environs/cloudspec"
 )
 
-// ValidatorsForModelMigrationSource returns a list of validators to run source controller for model migration,
+// ValidatorsForModelMigrationSource returns a list of validators to run source
+// controller for model migration.
+// Note: the target version can never be lower than the current version.
 func ValidatorsForModelMigrationSource(
 	targetVersion version.Number, cloudspec environscloudspec.CloudSpec,
 ) []Validator {
 	validators := []Validator{
 		getCheckTargetVersionForModel(targetVersion, MigrateToAllowed),
```

Comment:
I'm still not sure what this gives you, only to block you from moving to new versions.